### ISSUE TITLE
Additional animation triggers

### ISSUE THREAD

**Issue by HardeepAsrani** [Codeinwp/gutenberg-animation#5]
Now we load, by default, on the scroll. We can have an option to animate on load and on hover as well.

**Comment by webprom** [comment]
Hi!

If you load on scroll, when page reloaded not from top position, some top blocks remain not visible. When I use "animate.it" plugin i can insert their "on load" classes to the header and top of the page blocks, then i use "on scroll" classes below. Would be great to have it in your plugin too.

**Comment by HardeepAsrani** [comment]
@webprom Yes, it is. We are working on this plugin around this month so there will be many updates in coming weeks. Thank you for feedback!
